geth/common/utils.go

```diff
@@ -28,25 +27,10 @@ const (
 
 type contextKey string // in order to make sure that our context key does not collide with keys from other packages
 
-// errors
-var (
-	ErrInvalidAccountAddressOrKey = errors.New("cannot parse address or key to valid account address")
-)
-
-// ParseAccountString parses hex encoded string and returns is as accounts.Account.
-func ParseAccountString(account string) (accounts.Account, error) {
-	// valid address, convert to account
-	if common.IsHexAddress(account) {
-		return accounts.Account{Address: common.HexToAddress(account)}, nil
-	}
-
-	return accounts.Account{}, ErrInvalidAccountAddressOrKey
-}
-
 // FromAddress converts account address from string to common.Address.
 // The function is useful to format "From" field of send transaction struct.
 func FromAddress(accountAddress string) common.Address {
-	from, err := ParseAccountString(accountAddress)
+	from, err := account.ParseAccountString(accountAddress)
```

Comment:
That looks suspicious, that we have a package that `common` depends on. Maybe we should move these functions to `account` as well. When using it you add a transient dependency to `account` anyway.